src/app/App.js

```diff
@@ -18,8 +19,9 @@ class App extends Component {
     return (
       <main className="App">
         <h1>Rancid Tomatillos</h1>
+        {this.state.error && <h2>{this.state.error}</h2>}
         { this.state.displayMovieDetails && <MovieDetailsCard movie={this.state.displayMovieDetails} displayMovieLibrary={this.displayMovieLibrary} />}
```

Comment:
See my comment on line 59 - not sure how moving the API calls into a separate file will impact error handling.